io_ogre/ogre/mesh.py

```diff
@@ -579,7 +579,7 @@ def get_or_create_modifier(obj, modifier_name):
 
                 for vidx, v in enumerate(_remap_verts_):
                     pv = skey.data[ v.index ]
-                    x,y,z = swap( ob.matrix_world * pv.co - ob.matrix_world * v.co )
```

Comment:
is this change on purpose?

Reply:
hmmmmmmmmmmm .. I don't remember making this change ...